recipes/salib/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: salib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/SALib/SALib
+  git_tag: v{{ version }}
```

Comment:
This should include a chacksum (preferably sha256, but md5 can be used instead), I think.